app/views/deploys/new.html.erb

```diff
@@ -24,6 +24,17 @@
 
       <%= Samson::Hooks.render_views(:deploy_form, self, project: @project, form: form) %>
 
+      <% if @recent_releases %>
```

Comment:
Well, it's also kind of weird to show a label for "Recent Releases" and then show nothing if there are no recent releases. What do you think about that?